### DISCUSSION TITLE
DTU10MW controller

### DISCUSSION THREAD

**Discussion by yujooo** [NatLabRockies/ROSCO#127] [Q&A] [answered]
Hello,

I have a question about DTU10MW.yaml file in 2.4.0 version. (It was deleted in 2.5.0 version..)
Is this file a DTU10MW baseline controller model or just tunned file?

Thank you

**Comment by dzalkind** [comment, marked as answer]
Hi @yujooo,

We removed the DTU10MW.yaml file because we were having trouble keeping the OpenFAST model up to date and ensuring that the tuning input (yaml) we were providing was still working properly.  In the future, I could see us supporting the IEA 10MW turbine model because we could rely on that team to help with the OpenFAST modeling as it updates.

To answer your question, I would say that the tuning yaml for the DTU 10MW in 2.4.0 should work, but we can't make any promises about its performance and would recommend tuning it yourself to establish a good baseline: https://rosco.readthedocs.io/en/latest/source/standard_use.html#tuning-controllers-and-generating-discon-in. 

Happy to help with that process here.

Best, Dan

**Reply by yujooo** [reply]
Hello Dan,

Thank you so much for your kind reply.
Everything I was curious about was solved!

Best Regard
Yujoo
